packages/quip-apps-api/src/client.ts

```diff
@@ -165,6 +166,13 @@ export default class Client {
     public dateToPickValue: number = new Date().getTime();
     public displayWidthValue: number = 800;
     public documentMembersValue: User[] = [];
+    public documentTemplateSettingsValue: {
```

Comment:
can you define 
```
type DocumentTemplateSettings = {
    salesforce_org_id: string;
    salesforce_object_type: string;
};
```
somewhere, and then here you can do public documentTemplateSettingsValue: DocumentTemplateSettings = {
        salesforce_org_id: "",
        salesforce_object_type: ""
    }
instead?

Reply:
I could but I was following the same syntax as some of the other types. Do you still think I should make these changes?

Reply:
let's keep it